release-notes/1.0/1.0.19128.1-Preview.md

```diff
@@ -0,0 +1,139 @@
+# Release Notes
+
+## Microsoft.Data.SqlClient 1.0.19128.1 Preview released 8 May 2019
+
+This update adds the Always Encrypted APIs to the reference layer, exposing them appropriately based on the application target. No other fixes or changes were made in this preview from the previous preview.
+
+### Target Platform Support
+
+* .NET Framework 4.6+ (x64 only)
+* .NET Core 2.1+ (Windows x86, Windows x64, Linux, macOS)
+* .NET Standard 2.0+ (Windows x86, Windows x64, Linux, macOS)
+
+### Dependencies
+
+#### .NET Framework
+
+* System.Data.Common 4.3.0
+
+#### .NET Core
+
+* Microsoft.Win32.Registry 4.5.0
+* runtime.native.System.Data.SqlClient.sni 4.4.0
+* System.Security.Principal.Windows 4.5.0
+* System.Text.Encoding.CodePages 4.5.0
+* System.Configuration.ConfigurationManager 4.5.0
+
+#### .NET Standard
+
+* Microsoft.Win32.Registry 4.5.0
+* runtime.native.System.Data.SqlClient.sni 4.4.0
+* System.Buffers 4.4.0
+* System.Diagnostics.DiagnosticSource 4.5.0
+* System.Memory 4.5.1
+* System.Security.Principal.Windows 4.5.0
+* System.Text.Encoding.CodePages 4.5.0
+* System.Configuration.ConfigurationManager 4.5.0
+
+### New Features
+
+New features over .NET Framework 4.7.2 System.Data.SqlClient.
+
+* [Data Classification](#data-classification) - Available in Azure SQL Database and Microsoft SQL Server 2019 since CTP 2.0.
+* [UTF-8 support](#utf-8-support) - Available in Microsoft SQL Server SQL Server 2019 since CTP 2.3.
+
+New features over .NET Core 2.2 System.Data.SqlClient.
+
+* [Data Classification](#data-classification) - Available in Azure SQL Database and Microsoft SQL Server 2019 since CTP 2.0.
+* [UTF-8 support](#utf-8-support) - Available in Microsoft SQL Server SQL Server 2019 since CTP 2.3.
+* [Always Encrypted with Enclaves](#always-encrypted-with-enclaves) - Always Encrypted is available in Microsoft SQL Server 2016 and higher. Enclave support was introduced in Microsoft Sql Server 2019 CTP 2.0.
+
+### Fixed
+
+.NET Core
+
+* Always Encrypted APIs do not show up in Intellisense
+* Fixed in 1.0.19123.2

```

Comment:
Do we still need to keep fixes from 1.0.19123.2 in this release?

Reply:
I guess David wanted to keep them included - useful for GA maybe?

Reply:
OK let's keep it then.